samples/Plugin.FilePicker.Sample.Forms/Droid/Plugin.FilePicker.Sample.Forms.Droid.csproj

```diff
@@ -76,7 +76,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Xamarin.Forms">
-      <Version>3.3.0.912540</Version>
+      <Version>4.5.0.356</Version>

```

Comment:
Is this absolutely necessary? The lower the better, now we are forcing people to upgrade to 4.5 when they want to upgrade this

Reply:
It's only in the sample projects. The Plugin.FilePicker only references the SourceLink NuGet package.

Reply:
Oooo right 😴